src/tests/pokete/classes/poke/bank_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest.mock import patch
+from pokete.figure.bank import Bank
+
+with patch('os.get_terminal_size') as mock_get_terminal_size:
+    mock_get_terminal_size.return_value = (140, 40)
+    from pokete.figure.bank import Bank
+
+class bank_test(unittest.TestCase):
```

Comment:
```suggestion
class BankTest(unittest.TestCase):
```

Like in the other test files class naming should be in upper camel case and method naming in snake case.